core/docs/Input.md

```diff
@@ -306,6 +307,80 @@ Pairwise interactions are specified in meV per unique pair \<ij\>,
 while quadruplets are specified in meV per unique quadruplet \<ijkl\>.
 
 
+Micromagnetic Hamiltonian <a name="Micromagnetic"></a>
+--------------------------------------------------
+
+To use a micromagnetic Hamiltonian, use `micromagnetic` as input parameter after the `hamiltonian` keyword.
+
+Note that Spirit only supports rectilinear geometries when using this Hamiltonian.
+
+**Units:**
+
+In the micromagnetic model, Spirit uses SI units:
+- `m` (meter) for distances
+- `J` (Joule) for energies
+- `s` (second) for time
+
+Therefore,
+- `A [J/m]` for exchange stiffness
+- `D [J/m^2]` for DMI
+- `K [J/m^3]` for anisotropy
+- `Ms [A/m]` for saturation magnetisation
```

Comment:
Maybe we should add a section like this to the Heisenberg Hamiltonian documentation. IMO it it is quite useful to have this overview in the beginning